txwrapplog.py

```diff
@@ -22,8 +23,10 @@
 }
 
 
-def wrapp_observer(output):
+def wrapp_observer(output, service=None, host=None):
     json = jsonFileLogObserver(output, recordSeparator='')
+    service = service or os.environ.get('SERVICE_NAME')
```

Comment:
Why are you fetching the service here and also passing the information from the argument ?

Shouldn't this call be written outside the call to `wrapp_observer` ?